assets/stylesheets/index.scss

```diff
@@ -1,39 +1,43 @@
 @import "~@citizensadvice/design-system/scss/lib.scss";
 
-ul.landing-menu {
-  display: flex;
-  flex-wrap: wrap;
-  margin-bottom: 4rem;
+@import "./menu";
 
-  li {
-    width: 120px;
-    list-style-type: none;
-    text-align: center;
-    border: 1px transparent solid;
-    padding: 5px;
+.logo {
+  margin-top: $cads-spacing-7;
+  display: block;
+  background: url("/source/images/logo.png") no-repeat;
```

Comment:
Have you tested this in the built version? Just looking at this line, I would expect the built page to request this from `/intranet/source/images....`, which would be 404. Unless webpack knows to pull that out?

Reply:
Looking at the build CSS, webpack is converting the image into `data:img` so there isn't a request to the actual path, think it'll be OK